Orange/widgets/utils/colorpalette.py

```diff
@@ -597,66 +597,52 @@ def __getitem__(self, val):
 
 
 class ColorPaletteGenerator:
-    maxHueVal = 260
 
-    def __init__(self, number_of_colors=None, rgb_colors=DefaultRGBColors):
+    def __init__(self, number_of_colors=0, colors=DefaultRGBColors):
```

Comment:
Aye, I didn't. :flushed: Sorry to have put it on you like that. Thanks for reminding me so gently. Will surely strive to be exhaustive in the future.

Have reverted the parameter's name change and the mentioned widgets now work. Have also reconsidered the changed default value of the other parameter, but it doesn't seem to have an ill effect anywhere.

Would you recommend I revert all but the minimal required* changes (i.e. handling NaNs) or is the patch, in general, ok?

\* It's not really required. I could handle NaNs manually in my widget.